libs/vkd3d/vkd3d_private.h

```diff
@@ -618,8 +625,38 @@ struct d3d12_descriptor_heap
 
 HRESULT d3d12_descriptor_heap_create(struct d3d12_device *device,
         const D3D12_DESCRIPTOR_HEAP_DESC *desc, struct d3d12_descriptor_heap **descriptor_heap) DECLSPEC_HIDDEN;
+void d3d12_descriptor_heap_cleanup(struct d3d12_descriptor_heap *descriptor_heap) DECLSPEC_HIDDEN;
 struct d3d12_descriptor_heap *unsafe_impl_from_ID3D12DescriptorHeap(ID3D12DescriptorHeap *iface) DECLSPEC_HIDDEN;
 
+static inline unsigned int d3d12_descriptor_heap_sampler_set_index()
+{
+    return 0;
+}
+
+static inline unsigned int d3d12_descriptor_heap_cbv_set_index()
+{
+    return 0;
+}
+
+static inline unsigned int d3d12_descriptor_heap_srv_set_index(bool is_buffer)
+{
+    return 1 + (is_buffer ? 0 : 1);
+}
+
+static inline unsigned int d3d12_descriptor_heap_uav_set_index(bool is_buffer)
+{
+    return 3 + (is_buffer ? 0 : 1);
+}
+
+static inline uint32_t d3d12_desc_heap_offset(const struct d3d12_desc *dst)
+{
+    size_t heap_start = (size_t)dst->heap->descriptors;
+    return ((size_t)dst - heap_start) / sizeof(struct d3d12_desc);
```

Comment:
This is still here.

Reply:
Which commit is this? I thought this was later.

Reply:
7c20b0a3c4fed016444d8e433fa470000be0fd88

Reply:
Ok, found it, fixed.